docs/index-shared1.asciidoc

```diff
@@ -13,20 +13,12 @@ release-state can be: released | prerelease | unreleased
 
 :jdk:                   1.8.0
 :guide:                 https://www.elastic.co/guide/en/elasticsearch/guide/current/
-:ref:                   https://www.elastic.co/guide/en/elasticsearch/reference/{branch}/
-:xpack-ref:             https://www.elastic.co/guide/en/x-pack/{branch}/
-:kibana-ref:            https://www.elastic.co/guide/en/kibana/{branch}/
-:logstash:              https://www.elastic.co/guide/en/logstash/{branch}/
-:libbeat:               https://www.elastic.co/guide/en/beats/libbeat/{branch}/
-:filebeat:              https://www.elastic.co/guide/en/beats/filebeat/{branch}/
-:metricbeat:            https://www.elastic.co/guide/en/beats/metricbeat/{branch}/
+:libbeat:               https://www.elastic.co/guide/en/beats/libbeat/current/
```

Comment:
It looks like some of my changes got reverted. I want the paths to use {branch}.